### PR TITLE
Fix automation validation issues

### PR DESCRIPTION
**Pull request by will-byrne**
The validation passes for all three tests documents in the current import repo. When this is merged it should be tested in import with the current full dataset to see if there is anything we missed.

### CODE REVIEW

File: src/http-schema/http-schema.ts [hunk 1]
```diff
@@ -113,7 +120,7 @@ const PeerReviewSchema = Joi.object({
 });
 
 const AddressSchema = Joi.object({
-  addressCountry: Joi.string().required(),
+  addressCountry: Joi.string().optional(),
```
Comment (nlisgo) [inline src/http-schema/http-schema.ts:123]: We need to amend our types to reflect the optional status of this and some other properties

**Review by nlisgo — CHANGES_REQUESTED** [review]
Need to progress our types to reflect these validation changes or is this being treated as a followup concern?

File: src/http-schema/http-schema.ts (hunk 1 above)
Comment (will-byrne) [reply src/http-schema/http-schema.ts:123]: done

**Review by nlisgo — APPROVED** [review]
The changes I've requested have been done. I agree the full import can be done as a followup and we can raise validation errors as we encounter them.

**Pull request merged**